UN-OCHA/unocha-site
# UNO-516 Remove unused Image and Responsive Image styles

1. Rename Event image style for consistency and increase the width to match the max-width for content
2. Set to lazy instead of eager
3. Delete all unused Image and Responsive image styles

To test:
1. Import config, drush image:flush
2. Ensure no significant visual changes to existing images (*I will run VRT between dev and Preview for this)
